pocketbase/services/utils/crud_service.py

```diff
@@ -21,7 +21,7 @@ def get_list(
     ) -> ListResult:
         return self._get_list(self.base_crud_path(), page, per_page, query_params)
 
-    def _get_first_list_item(self, base_path: str, filter: str, query_params):
+    def get_first_list_item(self, filter: str, query_params):
```

Comment:
```suggestion
    def get_first_list_item(self, filter: str, query_params={}):
```
the query_params should be a named argument with empty dict default value to be consistent with the JS client